### PR TITLE
Ticket center Permission, Pagination, PageList, lastUpdateDate

### PR DESCRIPTION
**Pull request by JoschaO**
Added Permission based for ticketlists
Pagination as seperate input, that doesn't need max page from backend
Switch from Index to PageList for Daniel
Fixed lastUpdateDate 
Renamed ContactPage Service to user
Fixed Search to use right back end
Made a little more pretty

### CODE REVIEW

File: TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.ts [hunk 1]
```diff
@@ -12,30 +14,51 @@ import { NgbTabChangeEvent } from '@ng-bootstrap/ng-bootstrap/tabset/tabset';
...
 
-  constructor(private multiTicketService: MultiTicketService) {
+  constructor(private multiTicketService: MultiTicketService,
+              private userService: UserService) {
   }
 
   ngOnInit() {
-    this.getTicketList('');
-    this.ticketListResults.maxPages = 1;
+    console.warn('Gonna go get a perm. For the curls.');
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.ts]: This should probably not be here.

**Review by kixiQu — CHANGES_REQUESTED** [review]
There are some things here that I would probably do differently--the magic strings defining roles are an app setting on the back end, so having them built into the model files (rather than in app-settings or something) seems sketchy to me--but then again, we have to build that logic in, too. Maybe there are no good answers.

Anyway, delete the perm and I think it's good

**Review by ColinStClaire — DISMISSED** [review]
Looks good

**Review by ColinStClaire — DISMISSED** [review]
Looks good

**Review by kixiQu — APPROVED** [review]
yeeeeeee

**Pull request merged**